lib/galaxy/model/metadata.py

```diff
@@ -605,7 +605,16 @@ def wrap(self, value, session):
         if isinstance(value, int):
             return session.get(galaxy.model.MetadataFile, value)
         else:
-            return session.execute(select(galaxy.model.MetadataFile).filter_by(uuid=value)).scalar_one()
+            wrapped_value = session.query(galaxy.model.MetadataFile).filter_by(uuid=value).one_or_none()
```

Comment:
```suggestion
            wrapped_value = session.execute(select(galaxy.model.MetadataFile).filter_by(uuid=value)).scalar_one_or_none()
```
Let's keep the two statements identical (session.query... and session.execute...)